### DISCUSSION TITLE
Sending Logs from Zscaler

### DISCUSSION THREAD

**Discussion by sarim04** [Security-Onion-Solutions/securityonion#8528] [General] [answered]
Hello Team,
I am fairly new to security onion, I need some guidance on how to send logs from zscaler to security onion.

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/syslog.html:
> If you want to send syslog from other devices, you should check to see if the device has an existing [Filebeat](https://docs.securityonion.net/en/2.3/filebeat.html#filebeat) module at https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-modules.html. If so, using the [Filebeat](https://docs.securityonion.net/en/2.3/filebeat.html#filebeat) module should provide some parsing by default.

Filebeat has a Zscaler module:
https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-zscaler.html
